docs/segment-management-on-carbondata.md

```diff
@@ -54,6 +54,12 @@ concept which helps to maintain consistency of data and easy transaction managem
   SHOW SEGMENTS ON CarbonDatabase.CarbonTable
   ```
 
+  Show lastest 10 visible segments
```

Comment:
After compaction also, it should show latest segments right? For example, there are 6 segments, after major compaction, show segments with limit 2 will display 4.1 and 5th segment only, whereas latest segments are 0.2 and 0.3. I think we should get latest segments based on timestamp.

Reply:
Yeah. currently, the showed segments are ordered by loadname desc. in the type of：
+---+---------+-----------------------+
|ID |Status   |Load Start Time        |
+---+---------+-----------------------+
|5  |Compacted|2020-06-18 04:20:09.041|
|4.1|Success      |2020-06-18 04:20:09.041|
|4  |Compacted|2020-06-18 04:20:08.69 |
|3  |Compacted|2020-06-18 04:20:07.622|
|2.1|Compacted|2020-06-18 04:20:07.622|
|2  |Compacted|2020-06-18 04:20:07.226|

Reply:
yes. i think it is better to display latest segments based on timestamp on limit